components/Button.module.scss

```diff
@@ -26,4 +26,10 @@
   &:hover {
     box-shadow: 0 5px 15px rgba(0, 0, 0, 0.3);
   }
-}
+
+  &:disabled {
+    cursor: default;
```

Comment:
I like using `cursor: not-allowed` for this case.

Reply:
fixed this!